REPOSITORY_STANDARDS.md

```diff
@@ -0,0 +1,89 @@
+# Repository standards
+
+Here is a list of repository standards we have for our repositories. Much of the below is automated by our `dp-cli` tool for creating repositories but is documented here for completeness.
+
+## Contents
+
+- [naming conventions](#naming-conventions)
+- [visibility](#visibility)
+- [repository documentation](#repository-documentation)
+- [permissions](#permissions)
+
+## Naming conventions
+
+Repositories should be named with the prefix `dis-` to distinguish them as belonging to `Dissemination`. This helps to identify our repositories amongst the many that are part of ONSDigital.
+
+Historical repositories have the prefix `dp-` as they belonged to `Digital Publishing`.
+
+## Visibility
+
+We aim to work in the open where possible and so the majority of our repositories should be `public`.
+
+Repositories should be marked as `private` if they reveal details of:
+
+- ONS infrastructure
```

Comment:
This could be more explicit as my understanding is that any repos that contain IP ranges or other network details that would aid in an attacker need to be private so this generally means any repo that is used for provisioning a stack needs to be private. I believe that reusable and configurable terraform or ansible modules as well as other infrastructure components such as the dp-deployer or backup lambdas do not have this restriction and can be public.

I would suggest validating this with one of the SIRAs.

Reply:
ok, will leave as is at the moment and confirm to update in future PR.